### PR TITLE
Votable lists by friendsId

### PR DESCRIPTION
**Pull request by dotslash227**
- Gets the votable polls list by user in the friend list (if friend list is specified).

### CODE REVIEW

**Review by Hakeemmidan — COMMENTED** [review]
I got this error when going to the `/polls` page:
```
CastError: Cast to ObjectId failed for value "" at path "friendList" for model "polls"
```

**Comment by dotslash227** [comment]
> I got this error when going to the `/polls` page:
> 
> ```
> CastError: Cast to ObjectId failed for value "" at path "friendList" for model "polls"
> ```

Ok, i'll check that out.

**Comment by dotslash227** [comment]
> I got this error when going to the `/polls` page:
> 
> ```
> CastError: Cast to ObjectId failed for value "" at path "friendList" for model "polls"
> ```

Fixed. Please review it again.

File: server/routes/polls/polls.route.js [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
 const router = express.Router();
-
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+const ObjectId = Schema.ObjectId;
 const Poll = require('../../models/poll');
+const FriendList = require('../../models/friendList');
 const Vote = require('../../models/vote');
 const upload = require('./utils');
+// const parseArrOfFriendLists = require('./utils');
```
Comment (Hakeemmidan) [inline server/routes/polls/polls.route.js]:
```suggestion

```

**Review by Hakeemmidan — APPROVED** [review]

**Pull request merged**